src/main/java/org/datadog/jenkins/plugins/datadog/clients/DatadogHttpClient.java

```diff
@@ -160,7 +169,19 @@ public void validateConfiguration() throws IllegalArgumentException {
         }
 
         if (DatadogUtilities.getDatadogGlobalDescriptor().getEnableCiVisibility() ) {
-            logger.warning("Traces Collection only can be used if Datadog Agent reports to Datadog.");
+            if (webhookIntakeUrl == null || webhookIntakeUrl.isEmpty()) {
+                throw new IllegalArgumentException("Datadog Webhook Intake URL is not set properly");
+            }
+            try {
+                boolean webhookConnection = validateWebhookIntakeConnection(webhookIntakeUrl, apiKey);
+                if (!webhookConnection) {
+                    instance.setWebhookIntakeConnectionBroken(true);
+                    logger.warning("Connection broken, please double check both your Webhook Intake URL and Key");
```

Comment:
I see we also do this for the log intake connection, but do you mind making the log lines different so it's easier to debug?